src/app/views/users/_user.html.erb

```diff
@@ -11,8 +11,20 @@
           <%= link_to user.name, user_functions_path(user) %>
         </p>
         <p class="subtitle is-6">
-          @<%= user.username %>
+          <%= link_to user_github_account_url(user),  target: "_blank" do %>
+            @<%= user.username %>
+          <% end %>
         </p>
+        
+        <% unless current_user.nil? || user.id == current_user.id %> 
+          <% text = current_user.following?(user.id) ? "Unfollow" : "Follow" %>
+          <% method = current_user.following?(user.id) ? :delete : :post %>
+          <%= form_with model: current_user.followings.new(followee: user), method: method.to_sym, local: true do |form| %>
```

Comment:
use the same technique we're using for the like and saves, checkout the function footer view